docs/cody/capabilities/auto-edit.mdx

```diff
@@ -53,31 +89,56 @@ The following example demonstrates how to add Fireworks as an allowed LLM provid
 }
 ```
 
-## How does auto-edit work?
+</Tab>
 
-The system detects potential changes based on the following:
+<Tab title="JetBrains">
 
-- Cursor placement in the code
-- Recent changes made in similar contexts
-- Function usage patterns
-- Common refactoring patterns
+<Callout type="info">Auto-edit for JetBrains IDEs is Experimental and supports JetBrains versions 7.84.0+. It's available for Pro, Enterprise Starter, and Enterprise users on Cody Gateway. Auto-edit requires Fireworks to be enabled as a provider. Enterprise customers without Fireworks enabled can disable the feature flag.</Callout>
 
-## Auto-edit vs. Autocomplete
+## Enabling auto-edit in JetBrains
 
-In terms of feature-set and functionality, auto-edit and autocomplete have some key differences:
+You can opt-in the auto-edit feature from the JetBrains Cody plugin settings.
 
-|      **Feature**       |            **Autocomplete**            |                   **Auto-edit**                    |
-| ---------------------- | -------------------------------------- | -------------------------------------------------- |
-| **Text modification**  | Can only insert text, not delete       | Can both insert and delete text                    |
-| **Cursor position**    | Only proposes content after the cursor | Can modify code before and after the cursor        |
-| **Suggestion trigger** | Suggests on every character typed      | Proposes changes based on context and recent edits |
+- Click the three-dot menu on the top right of the Cody plugin window and select **Open Cody Settings Editor**
+- This will open the `cody_settings.json` file in your editor
+- Here, add `"cody.suggestions.mode": "auto-edit (Beta)"` setting to enable the auto-edit feature
```

Comment:
we're changing this to just `"auto-edit"`
here https://github.com/sourcegraph/cody/pull/7708/files